CHANGES/1033.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Normalize default ports according to :rfc:`3986`.
```

Comment:
Urgh.. I somehow missed that this was in imperative mood for some reason. It should refer to the change that has already happened since the previous release.

Reply:
```suggestion
The default protocol ports of well-known URI schemes are now taken into account
during the normalization of the URL string representation in accordance with
:rfc:`3986#section-3.2.3`.
```